rdd/node-microservices.md

```diff
@@ -0,0 +1,98 @@
+# Po.et Node Microservices Refactor
+
+It's time to finally break this guy down into proper microservices.
+
+## Background
+
+The current version of Po.et Node was built from the ground up with microservices in mind, but due to the business requirements at the time it was concieved, it was decided for it to be runnable as a single application, for greater ease of use.
```

Comment:
conceived is misspelled

How will ease of use change when we go to microservices? We don't want to make it more difficult to run a node.

Reply:
We are only officially supporting docker right now, so it won't change. It'll still be docker-compose up -d --build. Build time will be the same, too, since we'll only generate one docker image.

Reply:
With a slightly longer time to spin up the extra containers. We're talking seconds, not minutes.